metpy/plots/declarative.py

```diff
@@ -846,9 +849,10 @@ class ContourPlot(Plot2D):
 
     linecolor = Unicode('black')
```

Comment:
Oh, I guess `Unicode(<DEFAULT>)` works.

Reply:
Yep, that is what I used as an example, but I do like the None option, so I changed the colorbar above.